src/main/java/com/passpnu/passwordmanager/generator/StringPasswordGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.passpnu.passwordmanager.generator;
+
+import org.passay.CharacterRule;
+import org.passay.EnglishCharacterData;
+import org.passay.PasswordGenerator;
+import org.springframework.stereotype.Component;
+
+import java.security.SecureRandom;
+
+@Component
```

Comment:
Either make this `UtilityClass` and not `Component` or make this method non-static

Reply:
fixed